README.md

```diff
@@ -3,3 +3,28 @@
 # go-druid
 A Golang client for Druid.
 Now supports Query API and Common API.
+
+### Development
+
+#### Testing
+`go-druid` uses mage to run tests locally.
+    Install Mage: 
+```    
+    git clone https://github.com/magefile/mage
+    cd mage
+    go run bootstrap.go
+```
+`mage -l` provides a list of targets that can be run. Default is `Check`
+
+```
+Targets:
+  build            runs go mod download and then installs the binary.
+  check*           run linters and tests
+  fmt              run gofmt linter
+  lint             run golint linter https://github.com/golang/lint
+  testCoverHTML    generates test coverage report
+  testRace         run tests with race detector
```

Comment:
let's name it `test`

Reply:
original had both: https://github.com/gohugoio/hugo/blob/master/magefile.go#L188
If we keep all of this, why not keeping both ?

Reply:
would we ever need to run without race?

Reply:
let's keep both but default target should call `testRace`